source/application/REIXS/REIXSAppController.cpp

```diff
@@ -228,6 +227,7 @@ void REIXSAppController::shutdown() {
 
 	// Make sure we release/clean-up the beamline interface
 	AMBeamline::releaseBl();
+	AMStorageRing::releaseStorageRing();
```

Comment:
The above comment was made thinking I was looking at a different App Controller. The releaseStorageRing() call here should be removed, as it now lives in AMAppController::shutdown().